File: test_brute.js
const http = require('http');

async function test() {
  for(let i = 0; i < 20; i++) {
    const res = await fetch("http://127.0.0.1:3000/api/auth/login", {
      method: "POST",
      headers: { "Content-Type": "application/json" },
      body: JSON.stringify({
        username: "admin",
        password: "wrongpass" + i
      })
    });
    console.log(`[${i+1}] Request sent - Status: ${res.status} - Body: ${await res.text()}`);
  }
}
test();

File: app/applet/test_brute.py
import requests
import time

API_URL = "http://localhost:3000/api/auth/login"

for i in range(20):
    try:
        response = requests.post(API_URL, json={
            "username": "admin",
            "password": f"wrongpass{i}"
        }, timeout=2)
        print(f"[{i+1}] Request sent - Status: {response.status_code}")
        time.sleep(0.1)
    except Exception as e:
        print(f"[{i+1}] Error: {e}")
        break
